KsApi/GraphSchema.swift

```diff
@@ -400,13 +414,25 @@ extension Query.User: QueryType {
     case .optedOutOfRecommendations:            return "optedOutOfRecommendations"
     case let .savedProjects(args, fields):      return "savedProjects\(connection(args, fields))"
     case .showPublicProfile:                    return "showPublicProfile"
+    case let .storedCards(args, fields):        return "storedCards\(connection(args, fields))"
     case .slug:                                 return "slug"
     case .url:                                  return "url"
     case .userId:                               return "uid"
     }
   }
 }
 
+extension Query.User.CreditCard: QueryType {
+  public var description: String {
```

Comment:
If you make `CreditCard` be of type `String`, then here you can just do `self.rawValue`